joomla/joomla-cms
# [5.0] Update Codemirror to 6-th version

### Summary of Changes

Update Codemirror 6

I removed half of old parameters from plugin configuration.
But I think no one will notice.

### Testing Instructions

Apply patch, run `npm install`
Change default editor to codemirror, make sure editing still works


### Actual result BEFORE applying this Pull Request
Works


### Expected result AFTER applying this Pull Request
Works


### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: https://docs.joomla.org/Chunk4x:Extensions_Plugin_Manager_Edit_Editor_Group#Editor_-_CodeMirror
- [x] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/146


### Links
- https://codemirror.net/docs/migration/
